egs2/slurp/slu1/local/run_spm.sh

```diff
@@ -0,0 +1,40 @@
+# This script is called in data preparation step by local/data.sh
```

Comment:
This seems to be general? How about moving it to scripts/text ?

Reply:
Done!